src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -2296,15 +2322,8 @@ boolean function(Context context, DeclarationList declList) throws ParserExcepti
         if (info == null) {
             if (type.constructor) {
                 // get Info explicitly associated with all constructors
-                String name = dcl.cppName;
```

Comment:
This failed for instance in Pytorch for `c10::ArrayRef<decltype(::c10::impl::ScalarTypeToCPPType<::c10::ScalarType::Bool>::t)>` by returning `ScalarTypeToCPPType` instead of `ArrayRef`